README.md

```diff
@@ -1,6 +1,8 @@
 # Certificate Web UI
 
-[![Build Status](https://travis-ci.org/GovTechSG/certificate-web-ui.svg?branch=master)](https://travis-ci.org/GovTechSG/certificate-web-ui)
+[![Build Status]](https://github.com/OpenCerts/admin-website-v2/actions/workflows/ci.yml/badge.svg)
```

Comment:
add instruction to run integration tests locally

Reply:
issue fixed at #27